### ISSUE TITLE
What changed in 8.3/nightly from Jan 10 to Feb 2nd's RPM release?

### ISSUE THREAD

**Issue by worsco** [LMS-Community/slimserver#778]
I've been using lms 8.3 nightly on CentOS 7.9 for quite some time.

Since 1653836091 of 8.3.0, the size of the RPM is significantly reduced & the RPM nightlies released since then no longer work on EL 7.9.  

```
-rw-r--r--. 1 root root 206701833 Jan 10 13:03 logitechmediaserver-8.3.0-0.1.1641835163.noarch.rpm
-rw-r--r--. 1 root root 133653671 Feb  2 16:31 logitechmediaserver-8.3.0-0.1.1643836091.noarch.rpm
```
The version of perl on EL 7.9 is v5.16.3.

I suppose I can dump CONTENTS.cpio and compare what files are missing from the above two RPMs, or I can try installing the RPM on EL 8.5 (or 8.6 Beta since it has perl 5.32 available in the app streams).

@mherger Can you provide insight?

**Comment by mherger** [comment]
I indeed dropped support for older Perl versions at some point. It's not dropped completely, but the RPM would not longer come with any Perl older than 5.20... but I actually put 5.16 back in for CentOS 7 (see https://github.com/Logitech/slimserver-platforms/commit/3f0fcea337bb37461a3740366812d743a7da55a6). So... something else must have happened early February. Let me double check.

Can you confirm that a LMS installation would no longer have a CPAN/arch/5.16 sub-folder? What is the full output of `perl -v`?

**Comment by worsco** [comment]
Ok, I gave up on updating 8.3 nightly and didn't notice you put support back in.

```
# perl -v

This is perl 5, version 16, subversion 3 (v5.16.3) built for x86_64-linux-thread-multi
(with 44 registered patches, see perl -V for more detail)

Copyright 1987-2012, Larry Wall
<snip>
```

I just downloaded and installed `logitechmediaserver-8.3.0-0.1.1648408055.noarch.rpm` and it loaded w/o errors.

I'm closing this issue as CentOS 7 works.

**Comment by worsco** [comment]
PS: thank you so much for LMS!

**Comment by worsco** [comment]
BTW: pkgdiff was handy to find out what is different between the two RPMS: https://github.com/lvc/pkgdiff (and available as an RPM on EL 7.9).
